Cargo.toml

```diff
@@ -116,6 +116,7 @@ hyper = { version = "0.12", default-features = false, features = ["runtime"] }
 tokio-threadpool = "0.1.13"
 vlog = "0.1.4"
 twoway = "0.2.0"
+owning_ref = "0.4"
```

Comment:
oh, this lib is wonderful, maybe we can use to reduce the clone of Raft lib, @BusyJay

Reply:
https://github.com/pingcap/raft-rs/issues/245